script.js

```diff
@@ -44,7 +44,12 @@ async function searchByName(wantedName) {
     if (matching_ids.length === 1) {
         // Gets the game id
         const gameId = matching_ids[0]
-        const data = await getGameData(gameId);
+        const data = await getGameData(gameId)
+        if (!data.success) {
```

Comment:
This doesn't work for me locally (always says it failed).

It seems it should actually be `data[+gameId].success` (same for L74).

Reply:
Added a commit that fixed it: https://github.com/ScottRobinsonSky/MV-Week-4-Project/pull/5/commits/6915a3851906d8f193627334377bf77155d2951c